### PR TITLE
add docker files

### PR DESCRIPTION
**Pull request by yujinjcho**
run with

```
GOOGLE_CLIENT_ID={CLIENT_ID} \                                            
GOOGLE_CLIENT_SECRET={SECRET} \                                                                                        
JDBC_URL=jdbc:postgresql://host.docker.internal:5432/rcruitme \                                                                        
DB_USER=postgres \                                                                                                                     
  docker-compose up  
```

stop with `docker-compose down`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  client:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    volumes:
+      - "./:/rcruitme-front"
```
Comment (yujinjcho) [inline docker-compose.yml]: mounts your docker container to the current working dir. you can update src files and get hot reloading without having to rebuild image.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "rcruitme-front",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:9000",
+  "proxy": "http://rcruitme:9000",
```
Comment (yujinjcho) [inline package.json]: from docker-compose links field

**Comment by achiu8** [comment]
Are there corresponding changes that need to be made on the backend service? I'm getting errors about conflicting / allowed host names on the server side.
```
Host not allowed: rcruitme:9000
```

**Comment by yujinjcho** [comment]
oh my bad. made change [here](https://github.com/yujinjcho/rcruitme/pull/58)

**Pull request merged**